src/components/Garden/ProposalDetail/ProposalComments.tsx

```diff
@@ -2,7 +2,11 @@ import React from 'react'
 import { GU, Split, useLayout } from '@1hive/1hive-ui'
 import DiscourseComments from '@/components/DiscourseComments'
 
-function ProposalComments({ link }) {
+type ProposalComlinkmentsProps = {
```

Comment:
what mean `ProposalComlinkmentsProps` that part Comlinkments i dont get. Typo?

Reply:
Yep typo. I will fix it.

Reply:
Fixed